<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<arg name="quad"/>

	<arg name="goal_topic" default="goal"/>

	<!-- Start rmader commands -->
	<node ns="$(arg quad)" name="rmader_commands" pkg="rmader" type="rmader_commands.py" output="screen"/>

 	<!-- rmader -->
	<node ns="$(arg quad)" name="rmader" pkg="rmader" type="rmader_node" output="screen" clear_params="false" > 
    <rosparam file="$(find rmader)/param/rmader.yaml" command="load"/>
        <!-- Subscriptions -->
		<remap from="~state" to="state"/>
		<remap from="~who_plans" to="who_plans"/>
		<remap from="~term_goal" to="term_goal" />

		<!-- Publications -->
		<remap from="~traj" to="traj"/>
		<remap from="~goal" to="$(arg goal_topic)" />
		<remap from="~setpoint" to="setpoint" />

	</node>


</launch>